dind/start.sh

```diff
@@ -95,8 +95,45 @@ start_master ()
   docker tag k8s.gcr.io/kube-proxy:$(cat /docker_version) k8s.gcr.io/kube-proxy-amd64:$(cat /docker_version)
   docker tag k8s.gcr.io/kube-scheduler:$(cat /docker_version) k8s.gcr.io/kube-scheduler-amd64:$(cat /docker_version)
 
+  cat <<EOPOLICY > /etc/kubernetes/audit-policy.yaml
+apiVersion: audit.k8s.io/v1beta1
+kind: Policy
+omitStages:
+  - "RequestReceived"
+rules:
+- level: RequestResponse
+  resources:
+  - group: "" # core
+    resources: ["pods", "secrets"]
+  - group: "extensions"
+    resources: ["deployments"]
+EOPOLICY
+  chmod 0600 /etc/kubernetes/audit-policy.yaml
+  cat <<EOF > /etc/kubernetes/kubeadm.conf
+#apiServerExtraArgs:
+#  cert-extra-sans: $1
+apiVersion: kubeadm.k8s.io/v1alpha3
+kind: InitConfiguration
+kubernetesVersion: $(cat source_version | sed 's/^.//')
+auditPolicy:
+  logDir: /var/log/audit
+  logMaxAge: 2
+  path: /etc/kubernetes/audit-policy.yaml
+featureGates:
+  Auditing: true
+networking:
+  podSubnet: 192.168.0.0/16
+bootstrapTokens:
+- groups:
+  - system:bootstrappers:kubeadm:default-node-token
+  token: abcdef.abcdefghijklmnop
+  #token: tzc9vo.ugspyk7o3jxxhak6
```

Comment:
likewise, probably we should drop this line?

Reply:
Done